src/dropin-model.js

```diff
@@ -24,6 +25,10 @@ DropinModel.prototype = Object.create(EventEmitter.prototype, {
   constructor: DropinModel
 });
 
+DropinModel.prototype.isPaymentMethodRequestable = function () {
+  return Boolean(this._paymentMethodIsRequestable);
```

Comment:
For my own edification, is this preferrable to `return this._paymentMethodIsRequestable`?